File: BeFreeAPI/BeFreeAPI/Models/DDD.cs
namespace BeFreeAPI.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("tbDDD")]
    public partial class DDD
    {
        [Key]
        public int idDDD { get; set; }

        [StringLength(3)]
        public string codDDD { get; set; }

        public string descricao { get; set; }
    }
}

File: BeFree/app/src/main/java/tcc/befree/models/DDD.java
package tcc.befree.models;

/**
 * Created by gabro on 20/05/2017.
 */

public class DDD {
    public int id;
    public String codDDD;
    public String descricao;

    @Override
    public String toString(){
        return codDDD + " - " + descricao;
    }
}
